### PR TITLE
update contributing docs to reflect current practice and clarify steps

### PR DESCRIPTION
**Pull request by cewing**
This pull request clarifies the documentation for contributing to the documentation.  It also adds my name to the CONTRIBUTORS doc.

### CODE REVIEW

**Comment by bertjwregeer** [comment]
:+1: Looking good.

File: contributing.md [hunk 1]
```diff
@@ -56,11 +56,15 @@ System](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/install.ht
...
+2.  Change directories into the cloned repository
+
+         cd pyramid
+
 3.  Add a git remote "upstream" for the cloned fork.
 
          git remote add upstream git@github.com:Pylons/pyramid.git
 
-4.  Set an environment variable as instructed in the
+4.  Create a virtualenv and set an environment variable as instructed in the
```
Comment (stevepiercy) [inline contributing.md]: s/virtualenv/virtual environment

File: contributing.md [hunk 2]
```diff
@@ -69,28 +73,26 @@ System](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/install.ht
          # Windows
          set VENV=c:\hack-on-pyramid\env
 
-5.  Try to build the docs in your workspace.
+4.  Install `tox` into your virtualenv.
```
Comment (stevepiercy) [inline contributing.md]: s/virtualenv/virtual environment

**Comment by stevepiercy** [comment]
OK to merge as soon as build passes. I'll backport to 1.7 and 1.6. Thanks @cewing !

**Pull request merged**